crates/trycp_server/src/admin_interface.rs

```diff
@@ -0,0 +1,95 @@
+use serde::Serialize;
+use serde_derive::{Deserialize, Serialize};
+use serde_json::Value;
+
+use crate::rpc_util::internal_error;
+
+#[derive(Debug, Deserialize, Serialize)]
+#[serde(tag = "type")]
+enum AdminInterfaceMessage {
+    Request {
```

Comment:
Is there really a reason to have two different types here when the content is identical?

Reply:
I tried making a field `type: AdminInterfaceMessageType` but it would deserialize that way